src/SignalR/clients/java/signalr/core/signalr.client.java.core.javaproj

```diff
@@ -21,37 +19,10 @@
     <JavaBuildFiles Include="@(Jars);@(PomFile)"/>
   </ItemGroup>
 
-  <Target Name="Restore" />
-
-  <PropertyGroup>
-    <PackDependsOn Condition=" '$(NoBuild)' != 'true' ">
-      $(PackDependsOn);
-      Build
-    </PackDependsOn>
-  </PropertyGroup>
-
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.targets))\Directory.Build.targets" />
```

Comment:
nit: Can this be at the end of the file now, matching the usual conventions for projects that don't auto-import these files❔